step-maven-plugin/src/main/java/step/plugins/maven/DeployAutomationPackageMojo.java

```diff
@@ -52,6 +53,9 @@ public class DeployAutomationPackageMojo extends AbstractStepPluginMojo {
     @Parameter(property = "step.auth-token")
     private String authToken;
 
+    @Parameter(property = "step-deploy-automation-package.async")
+    private Boolean async;
```

Comment:
I'm not sure this name. The property is called 'async', but the real meaning is "don't wait for unlock, if the package is locked during update". But I don't know how to rename this property to make it clear...